pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -52,6 +52,7 @@
 , withIlbc ? withFullDeps
 , withJack ? withFullDeps && !stdenv.isDarwin # Jack audio
 , withLadspa ? withFullDeps # LADSPA audio filtering
+, withLibplacebo ? withFullDeps # libplacebo video processing library
```

Comment:
```suggestion
, withLibplacebo ? withFullDeps && !stdenv.isDarwin # libplacebo video processing library
```

```
Device creation failed: -12.
Failed to set value 'vulkan' for option 'init_hw_device': Cannot allocate memory
Error parsing global options: Cannot allocate memory
```

Reply:
Good point, Vulkan is not really a thing on Darwin... Added.